foodsaving/groups/tests/test_model.py

```diff
@@ -23,3 +23,16 @@ def test_roles_initialized(self):
         group = GroupFactory(members=[user])
         membership = GroupMembership.objects.filter(user=user, group=group).first()
         self.assertIn(roles.GROUP_MEMBERSHIP_MANAGER, membership.roles)
+
+    def test_active_member_count(self):
+        user = [UserFactory() for i in range(2)]
```

Comment:
I tend to write `_` if the variable is not used (instead of `i`)

Reply:
you are right, thanks. My python skills went a bit down...